src/pages/commits/ActiveCommits/BuildVariantCard.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Body } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariantDisplayName: string;
+}
+export const BuildVariantCard: React.FC<Props> = ({
+  buildVariantDisplayName,
+}) => <Card key={buildVariantDisplayName}>{buildVariantDisplayName}</Card>;
+
+const Card = styled(Body)`
```

Comment:
nit. Could we name this something else since this is a Text element and not really a [Card](https://www.mongodb.design/component/card/example/)

Reply:
Good idea! Does Label work?